### PR TITLE
Sanity clean-up after SpecimenType work.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
De-tabbed a half-dozen files that were heavily involved in SpecimenType changes, and renamed DeviceTypeHolder and all its methods to reflect that it now holds DeviceSpecimenTypes (a change which just seemed noisy in the context of the substantive changes it accompanies).

Much of this may become moot if we get auto-formatting in, but for now it's going to make life incrementally less annoying for a few of us.

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java [hunk 1]
```diff
@@ -22,80 +22,78 @@
...
     @JoinColumn(name = "default_device_specimen_type_id")
     private DeviceSpecimenType defaultDeviceSpecimen;
 
     @ManyToMany(fetch = FetchType.EAGER)
-    @JoinTable(
-            name = "facility_device_specimen_type",
-            joinColumns = @JoinColumn(name = "facility_id"),
-            inverseJoinColumns = @JoinColumn(name = "device_specimen_type_id")
-    )
+    @JoinTable(name = "facility_device_specimen_type", joinColumns = @JoinColumn(name = "facility_id"), inverseJoinColumns = @JoinColumn(name = "device_specimen_type_id"))
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java:49]: Oh, I don't like that change. Gonna stamp it out.

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java [hunk 2]
```diff
@@ -22,80 +22,78 @@
...
-			String phone, String email, Provider orderingProvider, DeviceSpecimenType defaultDeviceSpecimen,
-			List<DeviceSpecimenType> configuredDeviceSpecimens) {
-		super(org);
-		this.facilityName = facilityName;
-		this.cliaNumber = cliaNumber;
-		this.address = facilityAddress;
-		this.telephone = phone;
-		this.email = email;
-		this.orderingProvider = orderingProvider;
+    protected Facility() {
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/Facility.java:52]: I don't like this change either but I'm resigned to it 🤣

File: backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java [hunk 3]
```diff
@@ -112,7 +112,7 @@ public DeviceType createDeviceType(
         return dt;
     }
 
-    public DeviceTypeHolder getTypesForFacility(String defaultDeviceTypeId, List<String> configuredDeviceTypeIds) {
+    public DeviceSpecimenTypeHolder getTypesForFacility(String defaultDeviceTypeId, List<String> configuredDeviceTypeIds) {
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java:115]: Should change these argument names, too.

Comment (benwarfield-usds) [reply backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java:115]: Wait no I shouldn't LOL. Should rename the method and deprecate it.

File: backend/src/main/java/gov/cdc/usds/simplereport/service/model/DeviceSpecimenTypeHolder.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+public class DeviceSpecimenTypeHolder {
+
+    private DeviceSpecimenType _defaultDeviceType;
+    private List<DeviceSpecimenType> _allDeviceTypes;
+
+    public DeviceSpecimenTypeHolder(DeviceSpecimenType defaultType, List<DeviceSpecimenType> configuredTypes) {
+        super();
+        this._defaultDeviceType = defaultType;
+        this._allDeviceTypes = Collections.unmodifiableList(configuredTypes);
```
Comment (jeremyzitomer-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/model/DeviceSpecimenTypeHolder.java:16]: i suppose given the nature of this PR, this is not merely a nitpick: should be `_allDeviceSpecimenTypes`

**Review by jeremyzitomer-usds — APPROVED** [review]
just one comment in `DeviceSpecimenTypeHolder.java`, otherwise LGTM

File: backend/src/main/java/gov/cdc/usds/simplereport/service/model/DeviceSpecimenTypeHolder.java (hunk 4 above)
Comment (benwarfield-usds) [reply backend/src/main/java/gov/cdc/usds/simplereport/service/model/DeviceSpecimenTypeHolder.java:16]: That is precisely correct. 🤣

**Pull request merged**